### ISSUE TITLE
Disallow use of TRES format / save in a different way

### ISSUE THREAD

**Issue by jospic** [Zylann/godot_heightmap_module#6]
Hi, I'm not an expert, maybe I'm making some mistakes, but why the terrain not rendered during project execution?

In the editor is visible...
<img width="1440" alt="editor" src="https://user-images.githubusercontent.com/22050804/29355966-0db85254-8273-11e7-9d5c-fa65b6563f17.png">

but not when playing project...
<img width="1024" alt="run" src="https://user-images.githubusercontent.com/22050804/29356005-2a5486c6-8273-11e7-8820-d5a8f6e6fbab.png">

Thanks in advance.
-j

**Comment by Zylann** [comment]
Any errors in the console? Did you write a custom shader?

**Comment by jospic** [comment]
I'm not use a custom shader...I don't know shader language so good
This is console output:

![console](https://user-images.githubusercontent.com/22050804/29356580-fab55cfe-8274-11e7-8345-c5deeeb1cf28.png)

There are three error in debug panel, but I do not know if they are related to the problem

![debug](https://user-images.githubusercontent.com/22050804/29356715-70dfcef0-8275-11e7-8271-ee2344cc148b.png)

**Comment by Zylann** [comment]
They look unrelated. I'd need to test myself, something stupid might be going on

**Comment by Zylann** [comment]
It works fine for me:

![image](https://user-images.githubusercontent.com/1311555/29381609-2181fdba-82ca-11e7-995e-708ddde4aafa.png)

Could you provide an example project reproducing the bug?

**Comment by jospic** [comment]
Yes, this simple project DON'T work for me:

https://github.com/jospic/heightmap_test

I'm using a Godot 3 alpha older version of 4/5 days ago...But I could do a test with the last one.
Could be a graphics card issue? Or rendering preset?
Thanks in advance.
-j

**Comment by Zylann** [comment]
You didn't create a file for the heightmap. This is a usability issue, they just don't work otherwise.
Basically, just like you would not create Textures directly in the scene, you can't create a HeightMapData embedded in the scene, it needs to be saved in a `.heightmap` resource file (go to edit HeightMapData, click the save icon, then save to file).
Also, even if you could, the `.tscn` format is highly suboptimal to store such data^^"

**Comment by jospic** [comment]
Very well, it works! Many thanks, maybe, when you have time, you will do a short tutorial for beginners like me.
-j

**Comment by Mikerhinos** [comment]
You're right, saving the heightmap data as a .heightmap file did the trick, thanks ! :)
